tests/suites/test_suite_x509write.function

```diff
@@ -57,6 +57,7 @@ static int x509_crt_verifycsr( const unsigned char *buf, size_t buflen )
         return( MBEDTLS_ERR_X509_CERT_VERIFY_FAILED );
     }
 
+    mbedtls_x509_csr_free( &csr );
```

Comment:
This must also be called if an error occurs after `mbedtls_x509_csr_parse`.

Reply:
Thanks, fixed in dd7e134.